lib/js/jsre.nim

```diff
@@ -0,0 +1,42 @@
+## Regular Expressions for the JavaScript target.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+
+type RegExp* {.importjs.} = object    ## Regular Expressions for JavaScript target.
+  flags* {.importjs.}: cstring        ## cstring that contains the flags of the RegExp object.
+  dotAll* {.importjs.}: bool          ## Whether ``.`` matches newlines or not.
+  global* {.importjs.}: bool          ## Whether to test the regular expression against all possible matches in a string, or only against the first.
+  ignoreCase* {.importjs.}: bool      ## Whether to ignore case while attempting a match in a string.
+  multiline* {.importjs.}: bool       ## Whether or not to search in strings across multiple lines.
```

Comment:
Whether or not to search 
=>
Whether to search 

(simpler and not ambiguous)